docs/0install/setup.md

```diff
@@ -0,0 +1,28 @@
+The 0install distribution of the client is to allow zanata-cli to be available on multiple platform. This distribution requires [0install](http://0install.net/) to run.
+
+## Prerequisites
+
+1. Install [0install](http://0install.net/injector.html) on your machine.
```

Comment:
Cofirmed that with the jre dependency in place, Mac won't be able to run even if it has local valid JRE installed. So for now, I've commented out the dependency until they fix it in the 0install repo. That means the JRE installation will be applying to all OS for now.